Code/Framework/AzCore/Platform/Windows/AzCore/Debug/StackTracer_Windows.cpp

```diff
@@ -38,7 +38,7 @@ namespace AZ {
         struct SymbolStorageDynamicallyLoadedModules
         {
             size_t m_size;
-            DynamicallyLoadedModuleInfo m_modules[256];
+            DynamicallyLoadedModuleInfo m_modules[1028];
```

Comment:
What is this change for and why is it needed?
Also why was `1028` chosen?

Reply:
The stack size error that kept coming up, that was spamming the log like crazy. 1028 killed it.

Reply:
@amznestebanpapp I believe you are probably the person most familiar with the StackTracer(or at the very least the person who touched it last).
I am assuming the DynamicallyLoadedModuleInfo is for the number of different loaded dlls?

Reply:
are we really loading more than 256 dlls?

Yes, you are correct, this is used through the debug features in Windows to show stack traces. The DynamicallyLoadedModuleInfo is used mainly to prevent loading debug information of a dll multiple times. 

This sounds to me more of a case of registered modules not being treated properly and having duplicate ones (or maybe loading/unloading multiple times?)

@carlitosan How do you repro this message?